docs/splitter.md

```
@@ -0,0 +1,29 @@
+
+## IPv6
+
+The IPv6 splitter works identically and appears after an IPv6 result. For splits where the prefix difference is ≥ 63, the count is displayed as `2^N` to avoid integer overflow.
+
+## Export ASCII format
+
+```
+10.0.0.0/24
+├─ 10.0.0.0/26
+├─ 10.0.0.64/26
+├─ 10.0.0.128/26
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)